velox/substrait/README.md

```diff
@@ -0,0 +1,8 @@
+# Velox substrait upgrade
+
+Velox copy substrait proto files from `https://github.com/substrait-io/substrait/tree/main/proto/substrait` to generate substrait cpp files. So if you want to upgrade substrait to a certain version, follow the steps like below:
+
+- Copy substrait proto files for a certain version in `https://github.com/substrait-io/substrait/tree/main/proto/substrait` to velox substrait proto dir: `https://github.com/facebookincubator/velox/tree/main/velox/substrait/proto/substrait`
```

Comment:
How would someone know which version to use? Where do we store the current version used in Velox and how can we find out the latest version available in Substrait?

Reply:
Hi @mbasmanova, I can not find a better solution for you questions, because Substrait just add git tag for releasing a new version. Would you have some good suggestions?

Reply:
I don't know much about Substrait versioning. If they use git tags, that's fine. Let's just document that and provide instructions on how to look up the latest tag.

CC:  @vibhatha @westonpace

Reply:
Yes, Substrait uses git tags/releases.  So you can find out the latest version that way.  Eventually it would be nice for Velox to report the version it was using somewhere but until Substrait releases 1.0.0 I'm not sure how essential that is.